Sim_Harness_GUI/obj/Debug/Sim_Harness_GUI.gtk-gui.gui.stetic

```diff
@@ -7,22 +7,22 @@
     <widget-library name="glade-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
```

Comment:
This file is a byproduct of the build, and should never be committed. I am fairly confident I had made a PR that deletes this as well.